docker-compose.yml

```diff
@@ -9,4 +14,5 @@ services:
       - 4000:4000
       - 35729:35729
       - 3000:3000
-      -   80:4000
\ No newline at end of file
+      -   80:4000
+    command: jekyll serve --incremental
```

Comment:
I'm assuming that docker-compose is only used for development, but if this isn't the case I could be breaking things here

Reply:
correct it's optional dev-only stuff